mPowerSDK/Dashboard/APHDashboardViewController.m

```diff
@@ -308,22 +331,52 @@ - (APHScoring *)scoringForValueKey:(NSString *)valueKey
                                  latestOnly:NO];
 }
 
-- (void)prepareCorrelatedScoring{
-
-    self.correlatedScoring = [[APHScoring alloc] initWithTask:APHWalkingActivitySurveyIdentifier
-                                                 numberOfDays:-kNumberOfDaysToDisplay
-                                                     valueKey:kGaitScoreKey];
+- (void)prepareCorrelatedScoring
+{
+    self.correlatedScoring = nil;
+    self.correlatedScoring = self.correlatedScores[0].copy;
+    [self.correlatedScoring correlateWithScoringObject:self.correlatedScores[1].copy];
```

Comment:
If only ever two values, use firstObject and lastObject. Also, what happens if a user does not have use of both hands and only does the tapping test with either left or right?